docs/hopsml/experiment.rst

```diff
@@ -72,6 +72,19 @@ The path returned is to the root directory in Hopsworks of your project.
 
     ... Experiment code ...
 
+**Step 2 (optional)**. If you have your data in an s3 bucket you can read the data using TensorFlow by
```

Comment:
That is not required if the instance has a role that can read from S3, is that right? Maybe we could clarify that only if the instance does not have the role, they need to set the environment variables. 

Also isn't the experiments using hops-util-py which means that these variables are already set?

Reply:
But we said you should not use a single instance profile role. 

What do you mean by experiments using hops-util-py which means that these variables are already set?